lib/active_merchant/billing/gateways/decidir.rb

```diff
@@ -192,6 +192,17 @@ def valid_fraud_detection_option?(val)
         !val.nil? && val != ''
       end
 
+      # Returns a boolean if a boolean or 'true' or 'false' is sent in.
+      # Otherwise, returns the original value.
+      # This is so if the user sends an invalid boolean value like "okay",
+      # the gateway will send back an error.
+      def attempt_boolean(val)
```

Comment:
Any thoughts on a better name or comments for this method? I wanted it to be clear that it wasn't guaranteed to return a boolean value, and that that's okay.